config-connector/solutions/iam/kpt/service-account/iampolicy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicy
+metadata:
+  labels:
+    label-one: value-one
+  name: iampolicy-service-account # {"$ref":"#/definitions/io.k8s.cli.setters.iampolicy-name"}
+spec:
+  resourceRef:
+    apiVersion: iam.cnrm.cloud.google.com/v1beta1
+    kind: IAMServiceAccount
+    name: service-account-example # {"$ref":"#/definitions/io.k8s.cli.setters.service-account-name"}
+  bindings:
+  - role: roles/iam.serviceAccountKeyAdmin # {"$ref":"#/definitions/io.k8s.cli.substitutions.role-sub"}
+    members:
+    # Replace the ${PROJECT_ID?} below with your desired project ID.
+    - serviceAccount:service-account-example@${PROJECT_ID?}.iam.gserviceaccount.com # {"$ref":"#/definitions/io.k8s.cli.substitutions.project-id-in-sa-sub"}
```

Comment:
We want customers to have the freedom to use any members: user, group or service account. So we want customers to be able to set the entire field.

Details about the member field of IAM Policy can be found here: https://cloud.google.com/iam/docs/reference/rest/v1/Policy

Reply:
Discussed offline. New iampolicymember.ymal created. Please take a look. Thanks.